controllers/storagecluster/storagecluster_controller.go

```diff
@@ -235,8 +235,6 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			builder.WithPredicates(
```

Comment:
or should this use `util.ComposePredicates`? I don't remember does the default one allows any match or all matches.